_targets.R

```diff
@@ -0,0 +1,28 @@
+library(targets)
+library(dssecrets)
+library(tidyverse)
+
+options(tidyverse.quiet = TRUE)
+
+source("data-src/s3_upload.R")
+source("data-src/prep_logpile_data.R")
+
+list(
+  tar_target(
+    abbott_data,
+    read_csv('in/abbott-pools-and-fluxes.csv',
+             skip = 1,
+             col_types = c('ccccnnn'),
+             col_names = c('Type','Category','Feature_group','Feature','Vol_1000km3','Vol_km3', "Vol_m3"))
+  ),
+  tar_target(
+    water_volume,
+    prep_logpile_data(abbott_data, file_out = 'out/water_volume_logpile.csv')
```

Comment:
- [MUST DO] To function as a file target, you'll need to add `format = "file"` as a third argument to `tar_target()`. 
- [CAN DO] For following some DS best practices around file targets, we've been naming the target the same (or very similar) to the file that is actually saved (include `_[file extension]` but not the directories). So this target would be named `water_volume_logpile_csv`.